src/mame/heathkit/tlb.h

```diff
@@ -0,0 +1,119 @@
+// license:BSD-3-Clause
+// copyright-holders:Mark Garlanger
+/***************************************************************************
+
+    Heathkit Terminal Logic Board (TLB)
+
+****************************************************************************/
+
+#ifndef MAME_HEATHKIT_TLB_H
+#define MAME_HEATHKIT_TLB_H
+
+#pragma once
+
+#include "emu.h"
+#include "cpu/z80/z80.h"
+#include "machine/ins8250.h"
+#include "bus/rs232/rs232.h"
+#include "machine/mm5740.h"
+#include "sound/beep.h"
+#include "video/mc6845.h"
+#include "emupal.h"
+#include "screen.h"
+#include "speaker.h"
+
+class heath_tlb_device : public device_t
+{
+public:
+  heath_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+  // interface routines
+  auto serial_data_callback() { return write_sd.bind(); }
+  DECLARE_WRITE_LINE_MEMBER(cb1_w);
+  void init();
+
+protected:
+  heath_tlb_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock);
+
+  virtual ioport_constructor device_input_ports() const override;
+  virtual const tiny_rom_entry *device_rom_region() const override;
+  virtual void device_add_mconfig(machine_config &config) override;
+
+private:
+  virtual void device_start() override;
+
+  void mem_map(address_map &map);
+	void io_map(address_map &map);
+
+	void key_click_w(uint8_t data);
+	void bell_w(uint8_t data);
+	uint8_t kbd_key_r();
+	uint8_t kbd_flags_r();
+	DECLARE_READ_LINE_MEMBER(mm5740_shift_r);
+	DECLARE_READ_LINE_MEMBER(mm5740_control_r);
+	DECLARE_WRITE_LINE_MEMBER(mm5740_data_ready_w);
+
+	MC6845_UPDATE_ROW(crtc_update_row);
+
+	TIMER_CALLBACK_MEMBER(key_click_off);
+	TIMER_CALLBACK_MEMBER(bell_off);
+
+	emu_timer *m_key_click_timer = nullptr;
+	emu_timer *m_bell_timer = nullptr;
+
+  devcb_write_line write_sd;
```

Comment:
Please be consistent with `m_` prefixes – most of your data members have them, but `write_sd` lacks one.